src/d3a/setup/grid_fees/constant_grid_fees.py

```diff
@@ -20,9 +20,12 @@
 from d3a.models.strategy.load_hours import LoadHoursStrategy
 from d3a.models.appliance.pv import PVAppliance
 from d3a.models.strategy.pv import PVStrategy
+from d3a_interface.constants_limits import ConstSettings
 
 
 def get_setup(config):
+    ConstSettings.IAASettings.GRID_FEE_TYPE = 1
```

Comment:
Umm is it really needed even after setting fee type via `config.grid_fee_type` ?

Reply:
Indeed it's not really necessary. Removing it.